gapis/api/service.proto

```diff
@@ -238,3 +238,127 @@ message CubemapLevel {
 	image.Info negative_z = 5;
 	image.Info positive_z = 6;
 }
+
+message Metrics {
+  oneof metrics {
```

Comment:
Instead of having this as a `oneof`, I'd suggest having `memory_breakdown` as a regular field.
Then instead of having an enum for the single metrics you want, the interface can be a list of bool flags for the bits of info the client wants. That way the client can fetch everything it cares about in a single RPC.